### PR TITLE
ISSUE-56 | Trimming preactions and postactions before splitting to av…

### PR DESCRIPTION
**Pull request by meetchandan**
…oid executing empty sql

See Issue: https://github.com/spark-redshift-community/spark-redshift/issues/56

I think it is a very trivial fix but will save lot of time of developers facing the issue.
Please review and merge

Thanks :)

### CODE REVIEW

**Comment by meetchandan** [comment]
@lucagiovagnoli  Please take a look.

**Comment by meetchandan** [comment]
@lucagiovagnoli  Please take a look.

**Comment by lucagiovagnoli** [comment]
Hi @meetchandan, thanks a lot for your pull request! I've been travelling recently, will take a look soon.

File: src/main/scala/io/github/spark_redshift_community/spark/redshift/Parameters.scala [hunk 1]
```diff
@@ -229,7 +229,7 @@ private[redshift] object Parameters {
     /**
      * Extra options to append to the Redshift COPY command (e.g. "MAXERROR 100").
      */
-    def extraCopyOptions: String = parameters.get("extracopyoptions").getOrElse("")
+    def extraCopyOptions: String = parameters.getOrElse("extracopyoptions", "")
```
Comment (lucagiovagnoli) [inline src/main/scala/io/github/spark_redshift_community/spark/redshift/Parameters.scala]: Why did you change this? I think it was intended to be `get ('extracopyoptions')` and if it's not defined, just return an empty String

Comment (meetchandan) [reply src/main/scala/io/github/spark_redshift_community/spark/redshift/Parameters.scala]: It just more elegant way of writing. More cleaner I think.
The IDE was also giving hints to write it that way.
